xhr.bs

```diff
@@ -1042,12 +1039,9 @@ method must run these steps:
     <p>Let <var>response</var> be the result of
     <a for=/>fetching</a> <var>req</var>.
 
-    <p>If the {{XMLHttpRequest/timeout!!attribute}} attribute value is not
-    zero, <a for=fetch>terminate</a> this
-    <a for=/>fetch</a> if it has not returned
-    within the amount of milliseconds from the
-    {{XMLHttpRequest/timeout!!attribute}} attribute value with reason
-    <i>timeout</i>.
+    <p>If the {{XMLHttpRequest/timeout!!attribute}} attribute value is not zero, set the
+    <a>timed out flag</a> and <a for=fetch>terminate</a> <a for=/>fetching</a> if it has not
+    returned within the amount of milliseconds from the {{XMLHttpRequest/timeout!!attribute}}.
```

Comment:
I wonder if we should clarify what returned means here. But maybe as a follow-up since that's not your fault.